packages/components-react/projects/nextjs/pages/inline-notification.tsx

```diff
@@ -82,32 +82,63 @@ const InlineNotificationPage: NextPage = (): JSX.Element => {
         </PInlineNotification>
       </div>
 
-      <div className="playground light" title="should show inline-notification with action button">
+      <div className="playground light" title="should show inline-notification with action button on light background">
         <PInlineNotification
           heading="Some heading with action button"
           description="Some description"
           actionLabel="Some action label"
          />
       </div>
 
-      <div className="playground light" title="should show inline-notification with loading action button">
+      <div className="playground light" title="should show inline-notification with action button on dark background">
+        <PInlineNotification
+          heading="Some heading with action button"
+          description="Some description"
+          actionLabel="Some action label"
+          theme="dark"
+         />
+      </div>
+
+      <div className="playground light" title="should show inline-notification with loading action button on light background">
+        <PInlineNotification
+          heading="Some heading with action button"
+          description="Some description"
+          actionLabel="Some loading action label"
+          actionLoading={true}
+         />
+      </div>
+
+      <div className="playground light" title="should show inline-notification with loading action button on dark background">
         <PInlineNotification
           heading="Some heading with action button"
           description="Some description"
           actionLabel="Some loading action label"
           actionLoading={true}
+          theme="dark"
          />
       </div>
 
-      <div className="playground light" title="should show inline-notification in persistent mode">
+      <div className="playground light" title="should show inline-notification in persistent mode on light background">
         <PInlineNotification
           heading="Some persistent heading"
           description="Some description"
           persistent={true}
          />
       </div>
 
-      <div className="playground light" title="should show inline-notification in persistent mode with action button">
+      <div className="playground light" title="should show inline-notification in persistent mode on dark background">
```

Comment:
shouldn't it be "playground dark"? I think that's also the case in some other places.